src/Sylius/Behat/Resources/config/services/contexts/api.xml

```diff
@@ -33,6 +33,13 @@
             <argument type="service" id="session" />
         </service>
 
+        <service id="sylius.behat.context.api.admin.managing_product_reviews" class="Sylius\Behat\Context\Api\Admin\ManagingProductReviewsContext">
+            <argument>product_reviews</argument>
```

Comment:
IMO resource name we should put as an argument in service, it looks better than many magic strings in code.

Reply:
If you want to extract this string, I would rather vote on placing it on the client and create resource-specific clients. However, the idea is nice 👍

Reply:
If we decide to move into this direction we need an issue for that (or even better a pr, which will update all of currently existing resources)

Reply:
I will do this in separate PR